core/model/modx/processors/system/country/getlist.class.php

```diff
@@ -47,6 +44,16 @@ public function getCountryList()
                     unset($_country_lang[$key]);
                 }
             }
+            return $_country_lang;
+        }
+        $iso = $this->getProperty('iso', '');
```

Comment:
It is more a bug fix than a new feature. The empty array fix could be fixed different with an error message. The iso fix is an efficiency fix.